app/src/main/java/com/github/epfl/meili/profile/ProfileActivity.kt

```diff
@@ -64,7 +64,7 @@ class ProfileActivity : NavigableActivity(R.layout.activity_profile, R.id.profil
                 nameView.setText(Auth.getCurrentUser()!!.username)
             }
         }
-        viewModel.getRequestCreator().removeObservers(this)
+        viewModel.getRequestCreator().removeObservers(this) //TODO: why remove observers?? I want to reuse the view model in other parts of the code is it still fine?
```

Comment:
so that we don't have duplicate observers, because we might be adding observers multiple times in this activity

Reply:
Awesome! Thank you! I though we were removing all observers not just this.